cmd/completion.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(completionCmd)
+}
+
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate command line completion script",
+	Long: fmt.Sprintf(`To load completions:
+
+Bash:
+
+$ source <(%[1]s completion bash)
+
+# To load completions for each session, execute once:
```

Comment:
These are not comments, thus no need for #'s.

Reply:
Fixed